src/components/browse-data/files/FileTable.tsx

```diff
@@ -23,7 +23,7 @@ import {
 } from "../../../util/formatters";
 
 const fileQueryDefaults = {
-    page_size: 15
+    page_size: 50
```

Comment:
Unrelated to display size, but an increase to this limit has been requested for a while now.